lightstep-tracer-jre-bundle/assembly.xml

```diff
@@ -7,6 +7,12 @@
     <format>jar</format>
   </formats>
   <includeBaseDirectory>false</includeBaseDirectory>
+  <files>
+    <file>
+      <source>src/main/resources/META-INF/services/com.lightstep.tracer.shared.CollectorClientProvider</source>
+      <outputDirectory>META-INF/services</outputDirectory>
```

Comment:
to avoid overwriting of com.lightstep.tracer.shared.CollectorClientProvider by same file from tracer-okhttp and tracer-grpc